.teamcity/settings.kts

```diff
@@ -134,3 +135,31 @@ object PullRequests : BuildType({
         """.trimIndent()
     }
 })
+
+object VerifyPullRequests : BuildType({
+    templates(AbsoluteId("MinecraftForge_BuildPullRequests"), AbsoluteId("MinecraftForge_SetupGradleUtilsCiEnvironmen"), AbsoluteId("MinecraftForge_BuildWithDiscordNotifications"), AbsoluteId("MinecraftForge_SetupProjectUsingGradle"), AbsoluteId("MinecraftForge_BuildUsingGradle"))
+    id("MinecraftForge_MinecraftForge__PullRequestVerification")
+    name = "Pull Request Verification"
+    description = "Verifies pull requests for the project"
+
+    params {
+        checkbox("should_execute_build", "true", label = "Should build", description = "Indicates if the build task should be executed.", display = ParameterDisplay.HIDDEN,
+                checked = "true", unchecked = "false")
+        text(
+                "gradle_build_task",
+                "checkATs checkExcs checkSAS findFieldInstanceChecks checkLicenses checkPatches checkJarCompatibility",
```

Comment:
Ideally, the CI should only need to invoke the `check` task (possibly excluding any `test` tasks) and we should only need configure the buildscript such that our existing `check*` tasks are ran by `check`.